doc/superbuild/source/developers/style_guide/index.rst

```diff
@@ -14,15 +14,14 @@
 
 .. _Style:
 
-***********
```

Comment:
Add `===========` above (for some reason we can't suggest changes to a deleted line)